packages/web-client/app/components/card-pay/merchant-payment-request-card/index.hbs

```diff
@@ -0,0 +1,32 @@
+<Boxel::CardContainer
+  class="merchant-payment-request-card"
+  ...attributes
+>
+  <div class="merchant-payment-request-card__section merchant-payment-request-card__get-card-wallet">
+    <div class="merchant-payment-request-card__card-wallet">
+      {{svg-jar "card-wallet-app-icon" class="merchant-payment-request-card__card-wallet-icon" width="40" height="40"}}
+      <span class="merchant-payment-request-card__card-wallet-text">
+        Card Wallet
+      </span>
+    </div>
+    <a
+      class="merchant-payment-request-card__testflight-link"
+      href={{this.testFlightLink}}
+      target="_blank"
+      rel="noopener noreferrer"
+      data-test-payment-request-beta-access-link
+    >
+        Download TestFlight
+        {{svg-jar "caret-right-large" class="merchant-payment-request-card__testflight-link-icon"}}
+    </a>
```

Comment:
Could be done separately but what do you think of extracting this to a component that renders differently based on the environment? As I expect this will be replaced with an App Store button when out of beta, but the beta would still need to link to TestFlight

Reply:
I think extracting this early does help with limiting the surface area we need to change when we need a different rendering, if we are disciplined in how we use it and specify that it is for that purpose only.